guides/common/modules/proc_creating-a-virt-who-configuration.adoc

```diff
@@ -0,0 +1,156 @@
+[id="creating-a-virt-who-configuration_{context}"]
+= Creating a virt-who configuration
+
+You can create a virt-who configuration for {provider} by using the {ProjectWebUI}.
+
+To create a virt-who configuration by using the CLI, see xref:cli-creating-a-virt-who-configuration_{context}[].
+
+The virt-who configuration creates a `virt_who_reporter_[id]` user with the `Virt-who Reporter` role, which provides minimal permissions for virt-who reporting to {ProjectServer}.
```

Comment:
question: what does "[id]" mean/reference? It that a unique user ID? If so, maybe use italic emphasis if users have to replace it?

Reply:
Good questionI'm not sure whether the user would assign the ID or whether it is automatically generated. Will check.